### PR TITLE
DOCSP-36039: surface link to Realm Kotlin SDK

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-36039>
Staging:
- [Index](https://preview-mongodbrustagir.gatsbyjs.io/kotlin/DOCSP-36039-link-kt-sdk/#introduction)
- [FAQ](https://preview-mongodbrustagir.gatsbyjs.io/kotlin/DOCSP-36039-link-kt-sdk/faq/#what-is-the-difference-between-the-kotlin-driver-and-the-kotlin-sdk-)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -23,14 +23,20 @@ Introduction
...
 Welcome to the documentation site for the Kotlin Driver, the official
-MongoDB driver for server-side Kotlin applications using coroutines. Download it using
-`Maven <https://maven.apache.org/>`__ or
-`Gradle <https://gradle.org/>`__, or set up a runnable project by
-following our Quick Start guide.
-
-If your Kotlin application requires synchronous processing, use the
-:driver:`Sync Driver </kotlin-sync/>` which uses synchronous operations
-to make blocking calls to MongoDB.
+MongoDB driver for server-side Kotlin applications by using coroutines.
```
Comment (norareidy) [inline source/index.txt:26]: S: I think "by using" doesn't fully make sense here

```suggestion
MongoDB driver for server-side Kotlin applications that use coroutines.
```

File: source/index.txt [hunk 2]
```diff
@@ -23,14 +23,20 @@ Introduction
...
+<https://gradle.org/>`__, or set up a runnable project by following our
+Quick Start guide.
+
+.. tip:: Other Kotlin Platforms for MongoDB
+
+   If your Kotlin application requires synchronous processing, use the
+   :driver:`Sync Driver </kotlin-sync/>`, which uses synchronous operations
+   to make blocking calls to MongoDB.
+   
+   If you are developing an Android or Multiplatform application, you can
```
Comment (norareidy) [inline source/index.txt:37]: S: lowercase multiplatform (unless there's a specific reason it's capitalized)

```suggestion
   If you are developing an Android or multiplatform application, you can
```

**Review by norareidy — APPROVED** [review]
LGTM with a couple suggestions!

File: source/index.txt (hunk 2 above)
Comment (rustagir) [reply source/index.txt:37]: https://kotlinlang.org/docs/multiplatform.html based on this, it seems like Kotlin Multiplatform (KMP) is a proper noun. Making this more clear

**Comment by ashni-mongodb** [comment]
LGTM for coroutines! Could we do the same on the Kotlin sync landing page?

https://www.mongodb.com/docs/drivers/kotlin-sync/

**Pull request merged**